apps/web/src/screens/Dashboard/components/AptosTokensTab/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { type FC, useEffect, useState } from 'react';
+import { StyleSheet } from 'react-native';
+import type { TabAble } from '@walless/app';
+import { TabsHeader } from '@walless/app';
+import { aptosState } from '@walless/engine';
+import { getAptosConnection } from '@walless/engine/crawlers/aptos';
+import type { SlideOption } from '@walless/gui';
+import { Slider, View } from '@walless/gui';
+import { useSnapshot } from 'valtio';
+
+import DirectTransfer from './DirectTransfer';
+import OwnedTokens from './OwnedTokens';
+import PendingTokens from './PendingTokens';
+
+interface Props {
+	pubkey: string;
+}
+
+const AptosTokensTab: FC<Props> = ({ pubkey }) => {
+	const aptosSnap = useSnapshot(aptosState);
+	const [fee, setFee] = useState(0);
+
+	useEffect(() => {
+		const getFee = async () => {
+			const conn = await getAptosConnection();
+			const fee = await conn.estimateGasPrice();
+			return fee.gas_estimate / 10 ** 8;
+		};
+		getFee().then(setFee).catch(console.error);
```

Comment:
could try catch `setFee` inside `getFee` method instead of `.then().catch()`, then just run `getFee()`. You could ask @cloudle  for more detail